fulp_modules/Z_edits/antag_edits/changeling_spiders.dm

```diff
@@ -47,5 +47,5 @@
 	//We created a spider, let us announce this to ourselves.
 	ling_datum.spiders_created++
 
-//	new /obj/effect/mob_spawn/spider/bloody(user.loc) // Handled by Parent, so call ..() instead
+	//Parent handles new /obj/effect/mob_spawn/spider/bloody(user.loc)
```

Comment:
I'm pretty sure this is meant to look like parent so it's easier to spot

Reply:
So you think the old comment is better?

Reply:
Yea

Reply:
Agree